### PR TITLE
Feature/425 update

### PR DESCRIPTION
**Pull request by MaxPalmer-UH**

### CODE REVIEW

File: Source/LeapMotion/Private/LeapWrapper.cpp [hunk 1]
```diff
@@ -155,14 +168,16 @@ LEAP_TRACKING_EVENT* FLeapWrapper::GetInterpolatedFrameAtTime(int64 TimeStamp)
 		LeapInterpolateFrame(ConnectionHandle, TimeStamp, InterpolatedFrame, InterpolatedFrameSize);
 	}
 
-	return InterpolatedFrame;
+	//Disable interpolation test
```
Comment (MaxPalmer-UH) [inline Source/LeapMotion/Private/LeapWrapper.cpp:171]: @lazloringUH if we are not returning the interpolated frame then most of this code (or method) no longer makes sense?

File: Source/LeapMotion/Private/LeapWrapper.cpp [hunk 2]
```diff
@@ -315,32 +345,50 @@ void FLeapWrapper::HandleDeviceEvent(const LEAP_DEVICE_EVENT *DeviceEvent)
...
 			}
 		});
 	}
+	else {
+		free(DeviceProperties.serial);
+	}
 
-	free(DeviceProperties.serial);
-	LeapCloseDevice(DeviceHandle);
+	//LeapCloseDevice(DeviceHandle);
```
Comment (MaxPalmer-UH) [inline Source/LeapMotion/Private/LeapWrapper.cpp:376]: @lazloringUH are you happy to removed the commented out code or does it require more work?

File: Source/LeapMotion/Private/LeapWrapper.cpp [hunk 3]
```diff
@@ -315,32 +345,50 @@ void FLeapWrapper::HandleDeviceEvent(const LEAP_DEVICE_EVENT *DeviceEvent)
...
+	//TODO: maybe reverse. For multi device to we close?
 }
 
 /** Called by ServiceMessageLoop() when a device lost event is returned by LeapPollConnection(). */
 void FLeapWrapper::HandleDeviceLostEvent(const LEAP_DEVICE_EVENT *DeviceEvent) {
+	//todo: remove device handles matched here
```
Comment (MaxPalmer-UH) [inline Source/LeapMotion/Private/LeapWrapper.cpp:382]: Same as above, are you happy for this commented code to be removed?

**Review by lazloringUH — APPROVED** [review]
Tested on fresh pull, worked great!

**Pull request merged**